src/Resources/Resources.Test/ResourceGroups/SetAzureResourceGroupCommandTests.cs

```diff
@@ -71,7 +71,8 @@ public void UpdatesSetPSResourceGroupWithTag()
             PSResourceGroup expected = new PSResourceGroup()
             {
                 ResourceGroupName = expectedParameters.ResourceGroupName,
-                Tags = expectedParameters.Tag
+                Tags = expectedParameters.Tag,
+                ManagedBy = "test"
```

Comment:
@maddieclayton can we do the same testing to `ManagedBy` that we did for `ResourceGroupName` and `Tags` here?

- Add a private string for the value at the top of the file and use it here
- Add it to the `expectedParameters` object above
- Add an assertion below that the `ManagedBy` property is the same for `expectedParameters` and `actualParameters`